packages/angular-ui-components/angular.json

```diff
@@ -27,19 +27,15 @@
 					}
```

Comment:
we can add `"defaultConfiguration": "production"` to build configuration. With this we can remove from package.json the --configuration.